src/components/WalletAccounts.vue

```diff
@@ -71,10 +71,13 @@
         </ListItem>
         <div class="account-assets" :class="{ active: shouldExpandAccount(account) }">
           <ListItem
-            v-if="account.nfts && account.nfts.length > 0 && !isAssetList"
+            v-if="account.nfts && !isAssetList"
```

Comment:
@mpwanyi256 we need to also check that the NFT activity exists too.

Reply:
@Jennievon Ohkay... thanks for the catch

Reply:
@Jennievon Just pushed changes. Kindly take a look. thanks